src/commands/app/logs.ts

```diff
@@ -74,70 +76,113 @@ export default class AppLogs extends Command {
+
+    let maxSince: number;
+    switch (bundlePlanID) {
+      case 'free':
+        maxSince = now - 3600; // 1 hour
+        break;
+      case 'standard':
+        maxSince = now - 2592000; // 30 days
+        break;
+      case 'pro':
+        maxSince = now - 5184000; // 60 days
+        break;
+      default:
+        throw new Error('Unknown bundle plan type');
+    }
 
-      this.debug('Polling...');
+    const start: number = flags.since
+      ? this.getStart(`${flags.since}`)
+      : maxSince;
 
-      let logs: [string, string][] = [];
+    if (start < maxSince) {
+      console.error(
+        new Errors.CLIError(
+          BundlePlanError.max_logs_period(bundlePlanID),
+        ).render(),
+      );
+      process.exit(2);
+    }
 
-      try {
-        const data = await this.got(
-          `v2/projects/${project}/logs?start=${since}`,
-        ).json<ILog>();
+    let lastLogUnix: number | null = null;
 
-        logs = data.data[0].values;
-      } catch (error) {
-        if (error.response && error.response.statusCode === 404) {
-          // tslint:disable-next-line: no-console
-          console.error(new Errors.CLIError('App not found.').render());
-          process.exit(2);
-        }
-
-        if (error.response && error.response.statusCode === 428) {
-          const message = `To view more logs, upgrade your bundle plan, first. 
-Then try again.
-https://console.liara.ir/apps/${project}/resize`;
-
-          // tslint:disable-next-line: no-console
-          console.error(new Errors.CLIError(message).render());
-          process.exit(2);
-        }
-
-        this.debug(error.stack);
-      }
+    while (true) {
+      // Maybe find a better way of handling "lastLogUnix ?? 0"
+      const logs = await this.fetchLogs(
+        Math.max(start, lastLogUnix ?? 0),
+        project,
+      );
 
-      const lastLog = logs[0];
+      if (!logs?.length && !follow) {
+        break;
+      }
 
-      if (lastLog && lastLog[0] === 'Error') {
-        // tslint:disable-next-line: no-console
-        console.error(
-          new Errors.CLIError(`${lastLog[1]}
-Sorry for inconvenience. Please contact us.`).render(),
-        );
-        process.exit(1);
+      for (const log of logs) {
+        this.#printLogLine(log);
       }
 
+      const lastLog = logs[logs.length - 1];
+
       if (lastLog) {
         const unixTime = lastLog[0].slice(0, 10);
-        since = parseInt(unixTime) + 1;
+        lastLogUnix = parseInt(unixTime) + 1;
       }
+      await this.sleep(1000);
+    }
+  }
 
-      for (const log of logs.reverse()) {
-        this.#printLogLine(log);
+  fetchLogs = async (since: number, appName: string) => {
+    this.debug('Polling...');
+
+    try {
+      const url = `v2/projects/${appName}/logs`;
+      const data = await this.got(url, {
+        searchParams: {
+          start: since,
+          direction: 'forward',
+        },
+        timeout: {
+          request: 5000,
```

Comment:
```suggestion
          request: 10_000,
```

let's increase timeout